features/user_can_rate_recipes.feature

```diff
@@ -0,0 +1,32 @@
+Feature: User can rate recipes
+  As a user
+  In order to evaluate recipes
+  I would like to be able to rate recipes
+
+Background:
+  Given We have the following user:
+    | email           | password    |
+    | maran@test.com  | 12345678    |
+
+  Given We have the following recipes:
+    | title       	| description                 | ingredients        | directions              | user            |
+    | Meatball-Stew | This will make you sick     | Meat, onion, stuff | Stir it, mix it, eat it | maran@test.com  |
+
+  And I am logged in as "rater@test.com"
+
+@javascript
+Scenario: User can rate a recipe when visiting the recipe page
+  Given I visit the site
+  And I click "Meatball-Stew"
```

Comment:
We can use the available function "I am on the {string} recipe show page" instead of lines 19 & 20